tests/integration/run_tests.rs

```diff
@@ -428,6 +428,11 @@ itest!(env_file_missing {
   output: "run/env_file_missing.out",
 });
 
+itest!(env_file_multiple {
```

Comment:
Could I actually ask you to rewrite these three `env_file_` tests to spec tests? You can see some examples in `tests/specs/run/`

Reply:
Sure, will do so thank you for pointing it out.